specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -12,6 +12,64 @@ using TypeSpec.Versioning;
 
 namespace Azure.AI.Projects;
 
+
+@doc("Labeled results per row.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v1)
+model LabeledRows {
```

Comment:
This is the format of annotation eval, while the cloud eval row has much more information, I think we should converge a format which can hold more information?